Gruntfile.js

```diff
@@ -612,7 +644,8 @@ module.exports = function (grunt) {
         'replace:ternDefs',
         'requirejs:iframe',
         'exec:localize-dist',
-        'uglify'
+        'uglify',
+        'swPrecache'
```

Comment:
Wouldn't you want this to run after the `compress` task since production runs `grunt build-browser-compressed`?